iEmosoft_TestExecutioner/HelperObjects/AssertHelp.cs

```diff
@@ -38,6 +37,32 @@ public void AreNotEqual(object expected, object actual, string step)
                 throw;
             }
         }
+        public void AreNotSame(object expected, object actual, string step)
```

Comment:
What is the difference between AreSame and AreNotSame vs AreEqual and AreNotEqual?

Reply:
Oh, right... Those were not supposed to go in.. My bad!!